src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -250,10 +250,7 @@ extends AbstractSeq[T] with IndexedSeq[T] with Serializable {
       super.equals(other)
   }
 
-  override def toString() = {
-    val endStr = if (length > Range.MAX_PRINT) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("NumericRange(", ", ", endStr)
-  }
+  override def toString = "%d %s %d by %s".format(start, if (isInclusive) "to" else "until", end, step)
```

Comment:
I'm having trouble coming up with a format that makes sense for both BigInt and BigDecimal.

I've been playing around with the f interpolator and I've come up with this:
```scala
abstract class NumericRange[T]
  (val start: T, val end: T, val step: T, val isInclusive: Boolean)
  (implicit num: Integral[T])
...
override def toString = f"${num toDouble start}%.2f ${if (isInclusive) "to" else "until"} ${num toDouble end}%.2f by ${num toDouble step}%.2f"
```

But it feels wrong for BigInt to print out the decimal digits. Is there an easy way to know if the type contains Integral numbers?

Alternatively we could just print them as strings but that doesn't seem to format well (BigDecimal has way too many digits)

(It also seems odd that the class takes in an implicit num: Integral[T] when BigDecimal, etc aren't Integral but Fractional. Is this because NumericRange only needs an Integral[T] for its implementation?)